examples/nats-echo/main.go

```diff
@@ -33,8 +34,14 @@ import (
 // nats-echo -s demo.nats.io <subject>
 // nats-echo -s demo.nats.io:4443 <subject> (TLS version)
 
+const (
+	DefaultGeolocationURL = "https://ipapi.co/json"
+	GEOLOC_ENV_VARNAME    = "GEOLOCATION_JSON_URL"
```

Comment:
Would make it more of a Go name, GeolocationEnv

Reply:
Good call.  Fixed.